edx_exams/apps/api/v1/views.py

```diff
@@ -179,3 +180,20 @@ def patch(self, request, course_id):
             data = {"detail": "Invalid data", "errors": serializer.errors}
 
         return Response(status=response_status, data=data)
+
+
+class ProctoringProvidersView(ListAPIView):
+    """
+    Retrieve a list of all available proctoring providers
+
+    This endpoint returns an object for each provider
+
+    Path: /api/[version]/intents/most-recent-and-certain
+    Returns:
+     * 200: OK, list of ProctoringProviderObjects
+    """
+
+    authentication_classes = (JwtAuthentication,)
```

Comment:
So neat! 🎉

Reply:
Thanks for the tip! I didn't realize how simple it'd be haha 😅